source/help/settings/account-settings.md

```diff
@@ -1,151 +0,0 @@
-# Account Settings
-___
-Account Settings is accessible from the **Main Menu** by clicking the three dots at the top of the channels pane. From here, you can configure your profile settings, notification preferences, integrations, theme settings, and display options.
-
-## General
-Settings to configure name, username, nickname, email and profile picture.
-
-#### Full Name
-Full names appear in the Direct Messages member list and team management modal. By default, you will receive mention notifications when someone types your first name. Entering a full name is optional.
-
-#### Username
-Usernames are unique identifers appearing next to all posts. Pick something easy for teammates to recognize and recall. By default, you will receive mention notifications when someone types your username. In order to maintain message integrity, changing your username does not update @mentions in messages already posted.  
-
-#### Nickname
-Nicknames appear in the Direct Messages member list and team management modal. You will not receive mention notifications when someone types your nickname unless you add it to the *Words That Trigger Mentions* in **Account Settings > Notifications**.
-
-#### Email
-Email is used for sign-in, notifications, and password reset. Email requires verification if changed. If you are signing in using a single sign-on service, the email field is not editable and you will receive email notifications to the email you used to sign up to your SSO service.
-
-#### Profile Picture
-Profile pictures appear next to all posts and in the top of the channels pane next to your name. All users have a generic profile picture when they sign up for an account. Edit your profile picture by clicking **Select** and then choosing a picture in either JPG or PNG format that’s at least 128px wide and 128px high. For best results, choose an image that is square with the subject of interest centered. If you edit your profile picture, all past posts will appear with the new picture.
-
-## Security
-Settings to configure your password, view access history, and view or logout of active sessions.
-
-#### Password
-You may change your password if you’ve logged in by email. If you are signing in using a single sign-on service, the password field is not editable, and you must access your SSO service’s account settings to update your password.
-
-#### Multi-Factor Authentication 
-When this option is available you can requires a phone-based passcode in addition your password for sign-in. 
```

Comment:
Perhaps change this sentence to:
`When this option is available, a phone-based passcode is required in addition to your password for sign-in.`